### ISSUE TITLE
The label of Dataset 

### ISSUE THREAD

**Issue by ZZ00325** [vskadandale/vocalist#11]
Hello, in the "class Dataset(object)", there is the following piece of code for correctly selecting samples and assigning corresponding labels. I wonder if this is because the training data is unlabeled. Will assigning in this way lead to label assignment errors. If I know the positive and negative labels of the data set, can I directly modify the following code to assign the label to "y".

            if random.choice([True, False]):
                y = torch.ones(1).float()
                wav = pos_wav
            else:
                y = torch.zeros(1).float()
                try_counter = 0
                while True:
                    neg_frame_id = random.randint(interval_st, interval_end - v_context)
                    if neg_frame_id != pos_frame_id:
                        wav = self.get_wav(wavpath, neg_frame_id)
                        if rms_pos_wav > 0.01:
                            break
                        else:
                            if self.rms(wav) > 0.01 or try_counter>10:
                                break
                        try_counter += 1

                if try_counter > 10:
                    continue
            aud_tensor = torch.FloatTensor(wav)
